.github/workflows/build-and-deploy-meshery-extensions-playground.yml

```diff
@@ -68,22 +78,41 @@ jobs:
             },
           };
           EOF
-          
-      - name: Docker build & tag
-        working-directory: './meshery'
+
+      - name: Docker build
+        working-directory: "./meshery"
         run: |
           DOCKER_BUILDKIT=1 docker build -f install/docker/Dockerfile --no-cache -t layer5/meshery:kanvas-latest --build-arg TOKEN=${{ secrets.GLOBAL_TOKEN }} --build-arg GIT_COMMITSHA=${GITHUB_SHA::8} --build-arg GIT_VERSION=${{ env.MESHERY_VERSION }} --build-arg RELEASE_CHANNEL=${RELEASE_CHANNEL} .
 
+      - name: Bundle kanvas extension with the image
+        working-directory: 'meshery-extensions-packages'
+        run: |
+          wget https://github.com/layer5labs/meshery-extensions-packages/releases/download/${{ env.MESHERY_EXTENSION_VERSION }}/provider-meshery.tar.gz
+
+          mkdir extensions-package
+          tar xzf provider-meshery.tar.gz -C extensions-package 
+          mkdir -p provider/Meshery/${{ env.MESHERY_EXTENSION_VERSION }}
+          mv extensions-package provider/Meshery/${{ env.MESHERY_EXTENSION_VERSION }}
+          
+          ls;
+
+          docker run -d --name meshery-tmp layer5/meshery:kanvas-latest
+
+          docker cp ./Meshery meshery-tmp:/home/appuser/.meshery/provider/Meshery
+
+          docker container commit meshery-tmp layer5/meshery:kanvas-latest
+      
+      - name: Docker tag & push
+        run: |
+
           docker tag layer5/meshery:kanvas-latest layer5/meshery:kanvas-${GITHUB_SHA::8}
           docker tag layer5/meshery:kanvas-latest layer5/meshery:kanvas-${{ env.MESHERY_VERSION }}
-     
-      - name: Docker push
-        run: |
+
           docker push layer5/meshery:kanvas-latest
-          
+
           docker push layer5/meshery:kanvas-${GITHUB_SHA::8}
           docker push layer5/meshery:kanvas-${{ env.MESHERY_VERSION }}
-      
+
```

Comment:
The Docker-published GitHub Actions are much more full-featured and are quite reliable. Can we switch to them?

Use the multi-platform workflow as an example, please.